src/alaveteli.yaml

```diff
@@ -0,0 +1,36 @@
+---
+description: >
+  Platform for making public freedom of information requests to public bodies.
+  Alaveteli helps you lower the barriers that prevent citizens asking questions
+  of those in power.
+license: AGPL 3.0
+name: Alaveteli
+programmingLanguages:
+  - ruby
+rating: 5
+repoUrl: "https://github.com/mysociety/alaveteli"
+sdgs: [16]
+websiteUrl: "https://alaveteli.org/"
+
+contributionGuidelinesUrl: "https://alaveteli.org/docs/developers/"
+# They have no logo. We could otherwise use the mySociety logo
+logoUrl: "https://alaveteli.org/assets/img/favicon.png"
+
+otherLinks:
```

Comment:
Alternatively, we could simply add the list of all projects using Alaveteli: https://alaveteli.org/deployments/
This would probably be the most useful one.

Reply:
Since I added it in the description, I won't add it here to avoid duplication :)